flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/PipelinedSubpartitionWithReadViewTest.java

```diff
@@ -62,7 +64,8 @@
 
 	@Before
 	public void setup() throws IOException {
-		final ResultPartition parent = mock(ResultPartition.class);
+		final ResultPartition parent = PartitionTestUtils.createPartition(
+			ResultPartitionType.PIPELINED, NoOpFileChannelManager.INSTANCE, true, "LZ4", BUFFER_SIZE);
```

Comment:
Maybe it is better to make `isCompressed` as parameter, so all the previous existing tests can verify both cases?